### PR TITLE
fix: Add UTM params to all links in email templates

### PR DESCRIPTION
**Pull request by MrSltun**
This PR addresses [CX-3500], [CX-3502], [CX-3507], [CX-3508], [CX-3509], [CX-3510], [CX-3511], and [CX-3512]

This PR adds the UTM params to all the links in the submission email templates

[CX-3500]: https://artsyproduct.atlassian.net/browse/CX-3500?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3502]: https://artsyproduct.atlassian.net/browse/CX-3502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3507]: https://artsyproduct.atlassian.net/browse/CX-3507?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3508]: https://artsyproduct.atlassian.net/browse/CX-3508?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3509]: https://artsyproduct.atlassian.net/browse/CX-3509?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3510]: https://artsyproduct.atlassian.net/browse/CX-3510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3511]: https://artsyproduct.atlassian.net/browse/CX-3511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3512]: https://artsyproduct.atlassian.net/browse/CX-3512?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/partner_mailer/submission_digest.html.erb [hunk 1]
```diff
@@ -60,7 +60,7 @@
                       <table class='full-width-button'>
                         <tr>
                             <td class='submit-proposal-button'>
-                              <%= link_to('Send Proposal', Convection.config.artsy_cms_url) %>
+                              <%= link_to('Send Proposal', utm_url(Convection.config.artsy_cms_url, @utm_params)) %>
```
Comment (MrSltun) [inline app/views/partner_mailer/submission_digest.html.erb:63]: This function adds the UTM params to the cms link, should we keep?

File: app/views/user_mailer/other_submission_rejected.erb [hunk 2]
```diff
@@ -13,9 +13,9 @@
           <td class='email-sub-title'>
 
             <p>Dear <%= @submission.name %>,</p>
-            <p>Thank you for your interest in selling with <%= link_to 'Artsy', 'https://www.artsy.net', target: :_blank %> and for giving us a chance to consider your artwork.</p>
+            <p>Thank you for your interest in selling with <%= link_to 'Artsy', artsy_formatted_url('', @utm_params), target: :_blank %> and for giving us a chance to consider your artwork.</p>
             <p>Our specialists have reviewed your submission and while it’s a wonderful piece, we unfortunately don’t have demand for this artwork on Artsy at the moment.</p>
-            <p>If you’d like to know more, you can read about what our specialists are looking for in our <%= link_to 'Collector Help Center', 'https://support.artsy.net/hc/en-us/categories/360003689533-Sell', target: :_blank %></p>
+            <p>If you’d like to know more, you can read about what our specialists are looking for in our <%= link_to 'Collector Help Center', utm_url('https://support.artsy.net/hc/en-us/categories/360003689533-Sell', @utm_params), target: :_blank %></p>
```
Comment (MrSltun) [inline app/views/user_mailer/other_submission_rejected.erb:18]: This function adds the UTM params to the support.artsy.net link, should we keep it?

**Review by olerichter00 — APPROVED** [review]
Looks good 🌟

**Review by MounirDhahri — APPROVED** [review]
awesome. thanks Sultan

**Pull request merged**